condor/migration-guide.md

```diff
@@ -0,0 +1,193 @@
+
+Using 1.x Deploys in a 2.x Node:
+
+   - Casper 2.0 introduces a new Transaction model (replacing the old “Deploy”), but it will continue to accept and execute valid 1.x deploys for now​.
+   
+   - Casper Sidecar's `account_put_deploy` will be compatible with 1.x node `account_put_deploy`
+   
+   - This means the sidecar’s deploy submission method (`account_put_deploy`) remains usable with Casper 2.0 nodes, preserving compatibility for existing integration code. (In the Casper 2.0 JSON-RPC, this method is formally renamed to `account_put_transaction`​)
+   
+   - However, Casper Sidecar's `info_get_deploy` isn't 100% compatible with 1.x nodes `info_get_deploy`. 
+
+Therefore, Exchanges should review how they retrieve deploys/transactions and be aware of the response schema changes when querying for transaction status or details. For schema changes please refer the link [here](https://docs.casper.network/condor/jsonrpc#json-rpc-schema-definitions). 
+
+
+>[!IMPORTANT]
+> **Deprecation of 1.x Structures (e.g. Deploys)**
+> 
+> **1.x structures, such as Deploys, are being deprecated.** While they will continue to function in **Casper 2.0**, they are scheduled to be phased out.  
+> 
+> Exchanges should be aware of this evolution and prepare for the transition towards **TransactionV1** as the go-forward model.
+
+#### **Action Required**  
+
+➡️ Exchanges should review their integration and assess whether their systems depend on **info_get_deploy** results. If so, necessary updates must be made to ensure compatibility with **Casper 2.0**.
+
+➡️ To assess your next steps, please evaluate your integration against the 1.x -> 2.0 JSON RPC changes detailed in the link [here](https://docs.casper.network/condor/jsonrpc#json-rpc-schema-definitions). 
+
+Please refer to this [page](https://docs.casper.network/condor/jsonrpc) for more details on **casper-sidecar** and **JSON RPC** changes.
+
+### SDK Integration
+Depending on which SDK language (e.g. JavaScript, Go) you use, and which methods you interface with, you might not have to immediately migrate your integration. However, in order to be safe and future-proof, it is strongly recommended to upgrade your SDK to the latest version applicable to your stack. For most languages, this can be done immediately, as the SDKs maintain both forward (2.x) and backward (1.x) compatibility at the same time. 
+
+While it may not be strictly necessary to upgrade your integration if all you do is submit Deploys to the network (e.g. transfers in and out of accounts), it is STRONGLY RECOMMENDED to upgrade to the new SDK now, to experience a seamless upgrade to Casper 2.0 and beyond.
+
+>[!IMPORTANT]
+> **Addressable Entity is not enabled in Casper 2.0.**
+> 
+> Updating to a Casper 2.0-compatible SDK will be necessary when the **addressable entity** is enabled.
+> 
+> Once the **addressable entity** is enabled, exchanges will be required to use an SDK compatible with the Entity.
```

Comment:
We should not include not-enabled functionality in the migration guide. It unnecessarily overloads the reader w/o providing any clear value.